docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md

```diff
@@ -40,13 +40,15 @@ To perform the migration, you must first upgrade your installation to v2.20.1 or
 
 If your installation was installed via [Replicated](https://www.replicated.com/), use Replicated to upgrade your YugabyteDB Anywhere to a newer version.
 
+If you are upgrading a YugabyteDB Anywhere installation with high availability enabled, follow the instructions provided in [Upgrade instances](../../administer-yugabyte-platform/high-availability/#upgrade-instances).
+
 ## Upgrade using Replicated
 
 To start the upgrade, log in to the Replicated Admin Console via <https://:8800> and then perform the following:
 
 1. Navigate to **Dashboard** and click **View release history** to open **Release History**, as shown in the following illustration:
 
-  ![image](/images/yb-platform/upgrade-replicated1.png)
+    ![Release History](/images/yb-platform/upgrade-replicated1.png)
 
 1. Find the required release version and click the corresponding **Install**.
```

Comment:
Indent the text after this point. "If the required version is not in the list, cli....."